frontend/src/App.tsx

```diff
@@ -31,6 +36,16 @@ export default function App() {
         null
     );
 
+    const getUserInfo = () => {
+        setLoadingUserInfo(true);
+        axios
+            // .get("http://localhost:8081/userInfo")
+            .get("https://api.waroftheringcommunity.net:8080/userInfo")
+            .then((response) => setUserInfo(response.data))
+            .catch(console.error)
```

Comment:
Might we consider catching/handling specific errors here? By inspecting the status code of the response?
Just a thought. See https://axios-http.com/docs/handling_errors

Reply:
Absolutely, good call, done - and sorry for my scope creep 🤦‍♀️

Reply:
I isolated the scope creep bit to https://github.com/sirrus233/wotr-community-website/pull/173/commits/31510a89cd525369f92fe1e1d91ed2104f30836e, so it might be easier to review this commit-by-commit now!